docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,168 @@
+"""
+Backtracking on sloped terrain
+==============================
+
+Modeling backtracking for single-axis tracker arrays on sloped terrain.
+"""
+
+# %%
+# Tracker systems avoid row-to-row shading when the sun is low in the sky
+# by backtracking. The backtracking strategy orients the modules exactly
+# on the boundary between shaded and unshaded so that the modules are oriented
+# as much towards the sun as possible while still remaining unshaded.
+# Unlike the truetracking calculation (which only depends on solar position),
+# calculating the backtracking angle requires knowledge of the relative spacing
+# of adjacent tracker rows. This example shows how the backtracking angle
+# changes based on a vertical offset between rows caused by sloped terrain.
+# It uses :py:func:`pvlib.tracking.calc_axis_tilt` and
+# :py:func:`pvlib.tracking.calc_cross_axis_tilt` to calculate the necessary
+# array geometry parameters and :py:func:`pvlib.tracking.singleaxis` to
+# calculate the backtracking angles.
+#
+# Angle conventions
+# -----------------
+#
+# First let's go over the sign conventions used for angles. In contrast to
+# fixed-tilt arrays, the convention for the azimuth angle of a single-axis
+# tracker array is along the tracker axis. Note that the axis azimuth is
+# a property of the array and is distinct from the azimuth of the panel
+# orientation, which changes based on tracker angle.
+# Because the tracker axis points in two directions, there are two choices for
+# the axis azimuth angle, and by convention (at least in the northern
+# hemisphere), the more southward angle is chosen:
+#
+# .. image:: ../_images/tracker_azimuth_angle_convention.png
+#   :alt: Image showing the azimuth convention for single-axis tracker arrays.
+#   :width: 500
+#   :align: center
+#
+# Note that, as with fixed-tilt arrays, the axis azimuth is determined as the
+# angle clockwise from north.
+#
+# Using the axis azimuth convention above, the sign convention for tracker
+# rotations is given by the
+# `right-hand rule <https://en.wikipedia.org/wiki/Right-hand_rule>`_.
+# Point the right hand thumb along the axis in the direction of the axis
+# azimuth and the fingers curl in the direction of positive rotation angle:
+#
+# .. image:: ../_images/tracker_rotation_angle_convention.png
+#   :alt: Image showing the rotation sign convention for single-axis trackers.
+#   :width: 500
+#   :align: center
+#
+# So for an array with ``axis_azimuth=180`` (tracker axis aligned perfectly
+# north-south), pointing the right-hand thumb along the axis azimuth has the
+# fingers curling towards the west, meaning rotations towards the west are
+# positive and rotations towards the east are negative.
+#
+# The sign convention for ground slope follows the same convention -- align
+# the right-hand thumb along the tracker axis in the direction of the axis
```

Comment:
"the right-hand thumb along the tracker axis in the direction of the axis"

Did you mean along the ground slope azimuth?

Reply:
Okay, thanks for pointing this out.  I do mean along the tracker axis, but I did not mean "ground slope", I meant the cross-axis slope angle.  This text needs a revision.